packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -410,7 +415,29 @@ DumbTriggerManager.propTypes = {
    * Whether the vault will be closable or not.
    * @type {Boolean}
    */
-  vaultClosable: PropTypes.bool
+  vaultClosable: PropTypes.bool,
+  /**
+   *
+   */
+  vaultClient: PropTypes.object,
+  client: PropTypes.object,
```

Comment:
why not using `PropTypes.shape({ ... })` or `PropTypes.exact({ ... }) ` instead of object ?